src/btreemap/node/v1.rs

```diff
@@ -0,0 +1,152 @@
+use super::*;
```

Comment:
Is the whole of this file, just old methods for BoundedBTreeMap moved to one place?

Reply:
I wouldn't consider them old methods. I'd think of them more as shared functionality between both btrees.